src/spec/swagger-v11--submitted.json

```diff
@@ -718,6 +718,140 @@
         }
       }
     },
+    "/v1/trading/w/transfer": {
+      "post": {
+        "title": "Transfer",
+        "summary": "Submit a request for internal transfer (NOTE: currently not available to wider public).",
```

Comment:
Is this true? The functionality should be open to everyone now?

Reply:
Yes thats no longer true - probably its said that since we first released it when for the first few weeks it was only for use by us

Reply:
Understood, will remove it :)